code/Generators/DocBlockGenerator.php

```diff
@@ -9,7 +9,7 @@
  *
  * @package IDEAnnotator/Generators
  */
-class DocBlockGenerator
+class DocBlockGenerator extends Object
```

Comment:
Besides the extend calls, is there any reason to make it an object extension? It's adding a lot of overhead for an operation that probably won't be extended any time soon unless it's inside the module itself.

Any extended call should either be a PR or fixed inside the existing codebase instead of relying on the module.